config/config.go

```diff
@@ -56,5 +68,71 @@ func (c *Config) initConfigFromFile(path string) error {
 	if c.HaProxyPidFile == "" {
 		return errors.New("haproxy_pid_file is required")
 	}
+
+	if c.BackendTLS.CACertificatePath != "" {
+		pemData, err := os.ReadFile(c.BackendTLS.CACertificatePath)
+		if err != nil {
+			return err
+		}
+		for len(pemData) > 0 {
+			pemData = []byte(strings.TrimSpace(string(pemData)))
```

Comment:
Should we trim the contents before this for block? If there is only an empty string in there we don't want to fail.

Reply:
Will fix.